src/components/ConnectingTheDots/style.js

```diff
@@ -0,0 +1,58 @@
+import { css } from "@emotion/core"
+
+export const container = css`
+  margin: 0 auto;
+  padding-top: 25.9rem;
+  text-align: center;
+  max-width: 73.8rem;
+`
+export const titleStyle = theme => css`
+  margin: 0;
+  font-weight: 600;
+  font-family: ${theme.fonts.heading};
+  line-height: 7.6rem;
+`
+export const title1 = theme => css`
+  font-size: 6rem;
+  color: ${theme.colors.white};
+  letter-spacing: 2.7px;
+`
+export const title2 = theme => css`
+  font-size: 6.2rem;
+  color: ${theme.colors.greens.light};
+  letter-spacing: 0.4rem;
+  text-transform: uppercase;
+`
+export const subTitleStyle = theme => css`
+  margin: 0;
+  font-family: ${theme.fonts.body};
```

Comment:
By default all element's font-family is `theme.fonts.body`. Don't need to specify here